NetSecurity PowerShell reference update

TimShererWithAquent opened the pull request:
[1869908](https://dev.azure.com/mseng/TechnicalContent/_queries/edit/1869908/)

4 new cmdlets.

Changes to 8 others.

TimShererWithAquent committed:
Added cmdlets

TimShererWithAquent committed:
Cmdlet doc updates.

opbld32 commented:
Docs Build status updates of commit _[0a56570](https://github.com/TimShererWithAquent/windows-powershell-docs/commits/0a56570c4254328bd8ceb3bd92d835d8ac8d2434)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/copy-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/disable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/enable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/NetSecurity.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/NetSecurity.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/index?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/rename-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/set-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/update-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C8%5C24%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202108242031544534-2592%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=HJzQgmEXLAXF%2B6N%2FgI2mE6H6%2BZv5wWw182OWpUdcewo%3D&st=2021-08-24T20%3A33%3A29Z&se=2021-09-24T20%3A38%3A29Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

TimShererWithAquent committed:
Merge branch 'master' into netsec-v-tishe

opbld33 commented:
Docs Build status updates of commit _[b5409b2](https://github.com/TimShererWithAquent/windows-powershell-docs/commits/b5409b2130203398fd802669fcb76fe8c80d67e0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/copy-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/disable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/enable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/NetSecurity.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/NetSecurity.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/index?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/rename-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/set-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/update-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C8%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202109081559484476-2592%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=ZiWnz4aIxzaePFBvW%2BcmfAN1mWoO9HjZOwGTDP9De%2Bs%3D&st=2021-09-08T16%3A01%3A33Z&se=2021-10-09T16%3A06%3A33Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

TimShererWithAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/windows-powershell-docs into netsec-v-tishe

TimShererWithAquent committed:
Add content.

TimShererWithAquent committed:
Add content.

opbld32 commented:
Docs Build status updates of commit _[3e8f3f5](https://github.com/TimShererWithAquent/windows-powershell-docs/commits/3e8f3f58aff19071fb9920869857fdaca42854aa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/copy-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/disable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/enable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/NetSecurity.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/NetSecurity.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/index?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/rename-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/set-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/update-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C14%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202109141950130270-2592%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=w%2FV2%2B%2F3O1vPly5nOUkCJf%2BeTSOlz7rcxxxmzFtIVvmQ%3D&st=2021-09-14T19%3A51%3A47Z&se=2021-10-15T19%3A56%3A47Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

TimShererWithAquent committed:
Additional fixes.

TimShererWithAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/windows-powershell-docs into netsec-v-tishe

mstojens commented on docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md:
````diff
@@ -0,0 +1,198 @@
+
+### Example 1: Get a dynamic keyword address
+```powershell
+Get-NetFirewallDynamicKeywordAddress Id 258741ed-94a3-406b-9b0c-81fb145a4592 -PolicyStore ContosoPolicyStore
````
Is it normal to include a GUID in a parameter for an example command? Copy/pasting this will result in a failure because the customer's computer will not have that rule, but it is not immediately obvious that this should be a template (and they need to use a GUID from an existing rule of their own).

TimShererWithAquent replied:
I'll replace with "01234567-89ab-cdef-0123-456789abcdef", which is obviously fake. I don't remember why I used a GUID here - if I generated it or took it from an example. (The other approach is to use a place holder "<GUID>, but credible looking input is preferred.)

mstojens commented on docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md:
```diff
@@ -0,0 +1,212 @@
+
+This command creates an AutoResolve dynamic keyword address.
+
+Be sure to generate your own GUID.
```
If I leave the -Id parameter out, will a GUID be generated on my behalf? It seems odd to require a GUID be generated and then provided. If we truly require that, perhaps we should reference a GUID-generating cmdlet to use in-line here and elsewhere where we instruct the reader to bring their own GUID.

TimShererWithAquent replied:
@mstojens I got these two examples from here: https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-fasp/83c04df4-314b-444e-972a-85ff3c6e9726

The -id param is optional. Unless there's another way to identify a dynamic keyword address, it must get an ID somewhere. 

I don't want to say that I was guessing down in the param descriptions, but can you verify that it actually works that way?

mstojens replied:
Ok so I verified that the following will work:

> New-NetFirewallDynamicKeywordAddress -Id "{258741ED-94A3-406B-9B0C-81FB145A4592}" -Keyword "TestCmdlet" -AutoResolve $True

The GUID needs to be wrapped in quotes and curly brackets, Keyword need to be a string, and AutoResolve needs to use PowerShell boolean formats ($True, $False, 0, 1)

mstojens commented on docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md:
```diff
@@ -0,0 +1,212 @@
+
+### -Id
+Specifies a GUID for the dynamic keyword address.
+If you do not specify a GUID, the cmdlet creates one.
```
Ah, ok, cool. We should repeat this in the examples where we tell the reader to generate their own GUID. That section and this section give conflicting advice currently, and I think we should standardize on the easy behavior which is "leave the GUID generation to us" since GUIDs are inherently not human friendly and the reader is unlikely to have a GUID they prefer to another.

TimShererWithAquent replied:
I'm pulling the second line. You verified using a GUID in the example, so I'll go with that. I don't want to go beyond what I'm sure of.

mstojens commented on docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md:
````diff
@@ -0,0 +1,160 @@
+
+### Example 1: Update a dynamic keyword address
+```powershell
+Update-NetFirewallDynamicKeywordAddress Id 258741ed-94a3-406b-9b0c-81fb145a4592 -Address 10.0.0.15
````
Missing "-" in front of Id (on this line and more below)

mstojens commented on docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md:
````diff
@@ -0,0 +1,160 @@
+
+### Example 2: Append an address to a dynamic keyword address
+```powershell
+Update-NetFirewallDynamicKeywordAddress Id 258741ed-94a3-406b-9b0c-81fb145a4592 -Address 192.0.0.1 -Append $True
````
Here and in other examples you use "Address" when the documented flags and the defined cmdlet format both list only "Addresses". When playing with the cmdlet, it seems that "Address" is accepted in place of "Addresses". We should point  that out explicitly.

tojens-ietf reviewed (commented):
I have pointed out a few inconsistencies that will make the instructions clearer.

TimShererWithAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/windows-powershell-docs into netsec-v-tishe

TimShererWithAquent committed:
Add feedback, incomplete.

TimShererWithAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/windows-powershell-docs into netsec-v-tishe

TimShererWithAquent committed:
Incorporate feedback.

opbld33 commented:
Docs Build status updates of commit _[e5442b7](https://github.com/TimShererWithAquent/windows-powershell-docs/commits/e5442b7640ae5fc5b21dbe9baab7ce54347dc752)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/copy-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/disable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/enable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/NetSecurity.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/NetSecurity.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/index?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/rename-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/set-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/update-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C10%5C4%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202110041458562625-2592%5Cworkflow_report.html?sv=2020-08-04&se=2021-11-04T15%3A05%3A10Z&sr=b&sp=r&sig=PnUooCV%2FrYiPzsi9zSMm2W3ULoe%2F%2BzmWYwISckNWzZA%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

TimShererWithAquent committed:
Minor edits.

opbld32 commented:
Docs Build status updates of commit _[f961104](https://github.com/TimShererWithAquent/windows-powershell-docs/commits/f961104674c7eaeff69cf495542b428e81f0f5dc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Copy-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/copy-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Disable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/disable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Enable-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/enable-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Get-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/NetSecurity.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/NetSecurity.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/index?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Remove-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/remove-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Rename-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/rename-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Set-NetFirewallRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/set-netfirewallrule?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |
[docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/netsec-v-tishe/docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/netsecurity/update-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&branch=pr-en-us-2592) (WindowsServer2022-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C10%5C4%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202110041542153652-2592%5Cworkflow_report.html?sv=2020-08-04&se=2021-11-04T15%3A49%3A41Z&sr=b&sp=r&sig=ppBRDCW3B%2BWJSdlmVpX9qnsPsd88yPNYVzm4eB4zqR4%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

TimShererWithAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/windows-powershell-docs into netsec-v-tishe

TimShererWithAquent committed:
Changes to examples.

TimShererWithAquent committed:
Additional minor fixes.

TimShererWithAquent commented:
@mstojens - I've incorporated the changes from our email thread to **New-NetFirewallDynamicKeywordAddress**. I added spaces for clarity - and I don't think they'll break anything, but please have a look.

As I recall, that was the last outstanding issue. Any additional fixes are still welcome.

@PatAltimore for awareness.

TimShererWithAquent committed:
Merge branch 'master' into netsec-v-tishe

TimShererWithAquent committed:
Merge branch 'master' into netsec-v-tishe

dariomws reviewed (approved):
Minor suggestion.

TimShererWithAquent committed:
Apply suggestions from code review

Co-authored-by: Dario Woitasen <33589238+dariomws@users.noreply.github.com>

tiburd committed:
Merge branch 'master' into netsec-v-tishe
